pkg/profiles/proto.go

```diff
@@ -52,12 +53,20 @@ func protoToServiceProfile(parser *proto.Parser, namespace, name, clusterDomain
 				default:
 					path = fmt.Sprintf("/%s.%s/%s", pkg, service.Name, typed.Name)
 				}
+
+				rpc, ok := visitee.(*proto.RPC)
```

Comment:
I believe that if you use `typed` here instead of `visitee`, it will already be a `*proto.RPC` from the type-switch and we don't need to do a cast here.

Reply:
Yes, you are correct. The code will also be much simpler as the `ok` check also disappears.